### DISCUSSION TITLE
Mail is not working with ProtonMail Bridge

### DISCUSSION THREAD

**Discussion by houdini69** [firefly-iii/firefly-iii#6003] [General]
Hi, I'm trying to set up the mail account but I can't get it to work with ProtonMail Bridge (ProtonMail Bridge works on my server on Nextcloud).
> Possible email error: Unable to connect with STARTTLS: stream_socket_enable_crypto(): SSL operation failed with code 1. OpenSSL Error messages: error:1416F086:SSL routines:tls_process_server_certificate:certificate verify failed 

Here is how I set .env parameters for mail:

_MAIL_MAILER=smtp
MAIL_HOST=127.0.0.1
MAIL_PORT=1025
MAIL_FROM=xxxxxxxxx@pm.me
MAIL_USERNAME=xxxxxxxxxx
MAIL_PASSWORD=xxxxxxxxxxxxxxxx
MAIL_ENCRYPTION=null_
What's wrong with my setup?

**Comment by JC5** [comment]
Protonmail Bridge uses a self-signed certificate. See also https://github.com/firefly-iii/firefly-iii/issues/5996

**Reply by houdini69** [reply]
Ok. It mean I can't use ProtonMail Bridge with Firefly-iii?

**Reply by JC5** [reply]
That's not what I said. If you want it to work, you'll have to launch Firefly III with a reference to the self-signed certificate.

**Comment by houdini69** [comment]
Ok. I understand. I self-host Firefly-iii. So how to launch Firefly III with a reference to a self-sign certificate?

**Reply by JC5** [reply]
Depending on your server, there is usually a central set of root certificates your system uses. This is often `/etc/ssl/certs/cacert.pem` but it could be another file. Either way, add your self-signed certificate chain to this file, and it should work.

**Comment by houdini69** [comment]
Ok. I create self-signed certificate and add/update it to the central set of root certificates in my system (Debian) /etc/ssl/certs. However, I get the same error message.

**Reply by JC5** [reply]
Sorry, that's all the ideas I got. I hope you get it to work!

**Comment by houdini69** [comment]
Never mind. Thank you anyway. Firefly III works fine for me without email!
